### ISSUE TITLE
Get rebuild of the dogfood environment back into good working shape.

### ISSUE THREAD

**Issue by Berrescuda** [BenningtonCS/GFS#154]
So the system is currently working after the rebuild, but is still periodically plagued by mysterious bugs which need to be hammered out, specifically, getting the end to end test to succeed on the system.

**Comment by Berrescuda** [comment]
I... I think it might be okay? I don't want to say it's working because then it will break.

**Comment by Berrescuda** [comment]
It actually should be in good enough shape to port to AWS and maybe even run some tests.

**Comment by acencini** [comment]
:+1: 

what was wrong?

**Comment by Berrescuda** [comment]
There was a directory that was only partially camel cased on -some- of the pi's. (httpServerfiles instead of httpServerFiles - Queso Fiesta is having a little trouble with directories again, but I feel like @VulpesArtificem already fixed that somewhere else and we just have to find it and plug it in. Besides, we won't need to push or pull many changes to stuff in directories if all we want to do is run some tests and we're going to be booting up 50 identical machines anyway.)
I think I've got it under control and I just did an end to end test without problems, so we're looking good.
